plugins/json_importer.cc

```diff
@@ -89,6 +90,13 @@ JSONImporterConfigDescription()
              "Row name expression for output dataset. Note that each row "
              "must have a unique name and that names cannot be objects.",
              SqlExpression::parse("lineNumber()"));
+    addAuto("arrays", &JSONImporterConfig::arrays,
+            "Describes how arrays are encoded in the JSON output.  For "
+            "''parse' (default), the arrays become structured values. "
```

Comment:
This is copy pasted from https://github.com/mldbai/mldb/blob/master/sql/builtin_functions.cc#L1838

C'est la structure value description pour le parsing.

https://github.com/mldbai/mldb/blob/master/sql/expression_value.cc#L37